qtfred/ui/FredView.ui

```diff
@@ -20,7 +20,7 @@
      <x>0</x>
      <y>0</y>
      <width>926</width>
-     <height>28</height>
+     <height>21</height>
```

Comment:
No idea how the change to that line got there. Should I revert it?

Reply:
If you didn't make this change then you should revert that.